geode-core/build.gradle

```diff
@@ -301,3 +301,7 @@ distributedTest {
   // Some tests have inner tests that should be ignored
   exclude "**/*\$*.class"
 }
+
+afterEvaluate {
```

Comment:
Why does this need to be done in afterEvaluate?

Reply:
Turns out it does not. I had both generation tasks using identical formatting for consistency, but it is not needed here. I have removed the block for this dependency change.